users_permissions/README.md

```diff
@@ -81,6 +81,15 @@ Then run this command:
 1. Create a new app following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
     For the purpose of this sample you don't need to create an app from a template. You can just create an empty application.
 1. Click the button **Review draft & deploy**.
+1. Go to the **Sync** menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+    1. Don't create a schema. Skip by choosing "No thanks, continue to Sync".
+    1. Press the **Flexible Sync** button. Only Flexible Sync is supported in the Realm Flutter SDK.
+    1. Switch ON the [Development mode](https://www.mongodb.com/docs/atlas/app-services/sync/data-model/development-mode/) option.
+    1. Create a new database collection and choose a name for it.
+    1. Create a new queryable field used for setting App Service permissions. For our sample, the field is `owner_id`.
```

Comment:
Setting `owner_id` as queryable field is still needed because of the rules by `owner_id`. See https://jira.mongodb.org/browse/HELP-42805